### PR TITLE
infix functionality for properties get

### PR DESCRIPTION
**Pull request by Akyiaa**
story: [CLI: galasactl properties get does not support infixes#1577](https://github.com/galasa-dev/projectmanagement/issues/1577)

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-156-9jkmv for details. If you are unable to do so, please contact a member of the Galasa team.

File: pkg/properties/propertiesGet_test.go [hunk 1]
```diff
@@ -364,22 +514,270 @@ func TestValidNamespaceAndNoMatchingSuffixReturnsOk(t *testing.T) {
...
+Total:2
+`
 	//When
-	err := GetProperties(namespace, name, prefix, suffix, apiServerUrl, propertiesOutputFormat, console)
+	err := GetProperties(namespace, name, prefix, suffix, infix, apiServerUrl, propertiesOutputFormat, console)
 
 	//Then
 	assert.Nil(t, err)
 	assert.Equal(t, expectedOutput, console.ReadText())
 }
```
Comment (KirbyKatcher) [inline pkg/properties/propertiesGet_test.go]: These tests appear to be testing the API functionality not the CLI.
You should be able to delete them without impacting your testing coverage

**Review by KirbyKatcher — CHANGES_REQUESTED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-156-rklr4 for details. If you are unable to do so, please contact a member of the Galasa team.

File: pkg/cmd/propertiesGet.go [hunk 2]
```diff
@@ -36,13 +36,15 @@ var (
...
 	propertiesOutputFormat string
 )
 
 func init() {
 	formatters := properties.GetFormatterNamesString(properties.CreateFormatters())
-	propertiesGetCmd.PersistentFlags().StringVar(&propertiesPrefix, "prefix", "", "the name of properties from a specified namespace with the provided prefix")
-	propertiesGetCmd.PersistentFlags().StringVar(&propertiesSuffix, "suffix", "", "the name of properties from a specified namespace with the provided suffix")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesPrefix, "prefix", "", "returns properties from a specified namespace with the supplied prefix")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesSuffix, "suffix", "", "returns properties from a specified namespace with the supplied suffix")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesInfix, "infix", "", "returns properties from a specified namespace which contains the supplied infix or at least one of the supplied infixes if there are multiple")
```
Comment (KirbyKatcher) [inline pkg/cmd/propertiesGet.go:47]: Can we change these to the
```
Prefix to match against the start of the property name within the namespace
Infix(es) that could be part of the property name within the namespace, multiples are supplied as comma-separated list
Suffix to match against the end of the property name within the namespace
```

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-156-x2vlr for details. If you are unable to do so, please contact a member of the Galasa team.

File: pkg/cmd/propertiesGet.go [hunk 3]
```diff
@@ -36,13 +36,15 @@ var (
...
 	propertiesOutputFormat string
 )
 
 func init() {
 	formatters := properties.GetFormatterNamesString(properties.CreateFormatters())
-	propertiesGetCmd.PersistentFlags().StringVar(&propertiesPrefix, "prefix", "", "the name of properties from a specified namespace with the provided prefix")
-	propertiesGetCmd.PersistentFlags().StringVar(&propertiesSuffix, "suffix", "", "the name of properties from a specified namespace with the provided suffix")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesPrefix, "prefix", "", "Prefix to match against the start of the property name within the namespace")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesSuffix, "suffix", "", "Suffix to match against the end of the property name within the namespace")
+	propertiesGetCmd.PersistentFlags().StringVar(&propertiesInfix, "infix", "", "Infix(es) that could be part of the property name within the namespace, multiples are supplied as comma-separated list")
```
Comment (eamansour) [inline pkg/cmd/propertiesGet.go:47]: Do you think something like "... multiple infixes can be supplied ..." would read better than "... multiples are supplied..."?

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-156-n5n7l for details. If you are unable to do so, please contact a member of the Galasa team.

**Review by eamansour — APPROVED** [review]
Looks good, just one suggestion - approved!

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-156-cjs7v for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-156-hjlj5 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-156-g5r4c for details. If you are unable to do so, please contact a member of the Galasa team.

**Pull request merged**